jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1677,6 +1747,12 @@ ecma_builtin_typedarray_prototype_includes (ecma_typedarray_info_t *info_p, /**<
   {
     ecma_value_t element = getter_cb (info_p->buffer_p + search_pos);
 
+    if (ecma_arraybuffer_is_detached (info_p->array_buffer_p))
```

Comment:
According to the standard https://262.ecma-international.org/11.0/#sec-%typedarray%.prototype.fill this must be outsode of this loop.